node/src/blockchain/blockchain_interface.rs

```diff
@@ -1102,43 +1107,6 @@ mod tests {
         }
     }
 
-    #[test]
-    fn blockchain_interface_non_clandestine_can_request_both_eth_and_token_balances_happy_path() {
```

Comment:
I don't see this test migrated anywhere, is there any other test testing something similar?

Reply:
That's what I guessed. I just wanted to confirm that it is intended and not something that your eyes missed.